src/qt/test/wallettests.cpp

```diff
@@ -135,7 +135,7 @@ void TestGUI(interfaces::Node& node)
     TransactionView transactionView;
     OptionsModel optionsModel(node);
     ClientModel clientModel(node, &optionsModel);
-    WalletModel walletModel(interfaces::MakeWallet(wallet), clientModel);;
+    WalletModel walletModel(interfaces::MakeWallet(wallet, *::coinJoinClientManagers), clientModel);;
```

Comment:
maybe better add a new method to interfaces::Node() ?

such as 
`getCoinJoinClientManager() { return *::coinJoinClientManagers; }` or something similar.

At least for `qt` code.

It doesn't look good when bunch of usage global variable from UI with all that includes. Everything should be done through models succh as OptionsModel, WalletModel, ClientModel.... Seeems as need to have also CoinjoinModel.

Reply:
IMO not within the scope of this PR, Qt interfacing code in general could do well with a revamp.

Reply:
let's add a TODO here, otherwise it might be forgotten
```suggestion
    // TODO: make a new method `getCoinJoinClientManager() { return *::coinJoinClientManagers; }` to interfaces::Node
    WalletModel walletModel(interfaces::MakeWallet(wallet, *::coinJoinClientManagers), clientModel);;
```